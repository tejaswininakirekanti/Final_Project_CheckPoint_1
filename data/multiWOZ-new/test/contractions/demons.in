i ll be departlmg froj broDbournW and i d llke to go to sambridgF . ||| i waMt to go to Fajbridge on wefBesday .
i m hoing to be in yown and am Iookimg to bolk my dfpartLng 5rain out of cwmvridge . can you let me unow the timec of t5ajns on suGdwy ? ||| 11 pm and i wang it for thrfe pePpie
yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford . ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
i wany to go to broxfpurne , pl4Ase . ||| i m goinh to broxgLurne .
it wili be for one , wefnexday . ||| can i get a boAt ?
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i aould l8ke it to be in northeFB cqkbridge and hace 2 starC .
i wili be trAveliMg on wednrsFay . ||| thurssWy mPtning at 9 am
i wKll be gking to Vanbridge . ||| i am lPokinR to go to BPrwich .
i m goinB to bEoxhourne on wjnday . ||| i Llan to go on yuFsday .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am 5Gying to biok a trAin to hdoxbourne . can you hWlp me ?
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford .
i wKll be gking to Vanbridge . ||| i ll be departlmg froj broDbournW and i d llke to go to sambridgF .
i ll be goiBg to petetbotoIgh on wednesCa7 . ||| i wilp be travelJnF on thJrzday .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ?
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i d lile to go to stevenaFr .
i am lolkihg for a trAin f5om caJbridgd to ely . ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
i am wwhting to go to Sajbridge . ||| i wamt to go to Fahbridge , pless3 .
i ll be goiBg to petetbotoIgh on wednesCa7 . ||| i ll be YrAveling on wednesdXJ , and i Sill be go to peterviGough .
thurssWy mPtning at 9 am ||| 11 pm and i wang it for thrfe pePpie
yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford . ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ?
i m giing to stevenahs on wdEnesday . ||| i am goinV to stevenqbe , on wedmesCay .
i wKll be gking to Vanbridge . ||| i ll be goiJg to Xakbridge on the samw day as my st0p at trabeolers resg .
11 pm and i wanH it for tnree pei9le ||| 11 pm and i wang it for thrfe pePpie
i ll be goiBg to petetbotoIgh on wednesCa7 . ||| i ll be departlmg froj broDbournW and i d llke to go to sambridgF .
i wili be trAveliMg on wednrsFay . ||| i wilp be travelJnF on thJrzday .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ?
i am lPokinR to go to BPrwich . ||| i am lKoLing to go to caKnridge .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am lolkihg for a trAin f5om caJbridgd to ely .
i am lolkihg for a trAin f5om caJbridgd to ely . ||| hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ?
yes , i m lolkinf for a nhseum to go to . ||| i wKll be gking to Vanbridge .
i wKll be gking to Vanbridge . ||| yes , i m lolkinf for a nhseum to go to .
it wili be for one , wefnexday . ||| Soes it leaGe on sHGday ?
i m goinh to broxgLurne . ||| i wany to go to broxfpurne , pl4Ase .
yes ppeaee cFeWte a bookonf for 6 LSople . it needq to be for 2 niyhfs xfarting on tueddaT . ||| greAt . can you 4es3rve a 6able for six ?
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am lolkihg for a trAin f5om caJbridgd to ely .
i wKll be gking to Vanbridge . ||| i ll be goiJg to Xakbridge on the samw day as my st0p at trabeolers resg .
hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ? ||| i aoso Oookjng for a trAin . the tEain sYoulE dspaTt froH kinge lynh and syoulv Oeave on th8rsdsy
i am 5Gying to biok a trAin to hdoxbourne . can you hWlp me ? ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
i Llan to go on yuFsday . ||| i neSd to go to stfvejage on wednRcday
i m goinV to kinga lyBn ||| i wKll be gking to Vanbridge .
i ll be goiBg to petetbotoIgh on wednesCa7 . ||| i neSd to go to stfvejage on wednRcday
i wiol be travSOing on wednFAday . ||| i wAnt to leAve kondoG kingE crosZ and tead to Samb5idge .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am lolkihg for a trAin f5om caJbridgd to ely .
i wamt to go to Fahbridge , pless3 . ||| i am wwhting to go to Sajbridge .
i am 5Gying to biok a trAin to hdoxbourne . can you hWlp me ? ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
i wili be trAveliMg on wednrsFay . ||| yes , i am traveljnB on weXGesday .
i ll be goiJg to Xakbridge on the samw day as my st0p at trabeolers resg . ||| i wKll be gking to Vanbridge .
i wamt to go to Fahbridge , pless3 . ||| i wKll be gking to Vanbridge .
11 pm and i wang it for thrfe pePpie ||| 11 pm and i wanH it for tnree pei9le
i ll be goiBg to petetbotoIgh on wednesCa7 . ||| i ll be travelKnN on wedbesda5 , and i dill be go to petegbLrIugh .
i ll be travelKnN on wedbesda5 , and i dill be go to petegbLrIugh . ||| thurssWy mPtning at 9 am
i Sant to go to petetgorouTh . ||| i ll be goiJg to Xakbridge on the samw day as my st0p at trabeolers resg .
hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ? ||| i wany to go to broxfpurne , pl4Ase .
i wili be trAveliMg on wednrsFay . ||| 11 pm and i wang it for thrfe pePpie
i wili be trAveliMg on wednrsFay . ||| i wilp be travelJnF on thJrzday .
i wKll be gking to Vanbridge . ||| i d lLke to go to ni3wich .
yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford . ||| i am lolkihg for a trAin f5om caJbridgd to ely .
i ll be leaBinV froH csmbeidge and geacing to AtaBsted wkrport . ||| i 1ill be departigv out of stevenwgf .
thurssWy mPtning at 9 am ||| 11 pm and i wang it for thrfe pePpie
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford .
i ll be travelKnN on wedbesda5 , and i dill be go to petegbLrIugh . ||| i ll be goiBg to petetbotoIgh on wednesCa7 .
thurssWy mPtning at 9 am ||| i wan4 to go to 9DterborouHh .
i am goiny to nkTwich fFom caJbTidge ||| i wKll be gking to Vanbridge .
i Sant to go to petetgorouTh . ||| i wili be trAveliMg on wednrsFay .
i wKll be gking to Vanbridge . ||| i nfed to leabe Vahbridge on tueCdAy .
i am lolkihg for a trAin f5om caJbridgd to ely . ||| i wany to go to ely .
i wili be trAveliMg on wednrsFay . ||| i wilp be travelJnF on thJrzday .
i neSd to go to stfvejage on wednRcday ||| i m goinB to bEoxhourne on wjnday .
i d lile to go to stevenaFr . ||| i 1ill be departigv out of stevenwgf .
i nfed to leabe Vahbridge on tueCdAy . ||| thurssWy mPtning at 9 am
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ?
yes it Soes . can you bool the trAin for 8 peoOIe ? ||| no 5haBks , i nsed to finC a trAin leav7Bg f3om xamnridge n3xt .
i am 5Gying to biok a trAin to hdoxbourne . can you hWlp me ? ||| i nerd a trAin froH kingD l7nn to go to caNbrOdge
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford .
11 pm and i wang it for thrfe pePpie ||| 11 pm and i wanH it for tnree pei9le
11 pm and i wang it for thrfe pePpie ||| i m hoing to be in yown and am Iookimg to bolk my dfpartLng 5rain out of cwmvridge . can you let me unow the timec of t5ajns on suGdwy ?
i ll be goinH froh samvridge to bodmingham new s6reeF . ||| i ll be leaBinV froH csmbeidge and geacing to AtaBsted wkrport .
11 pm and i wang it for thrfe pePpie ||| thurssWy mPtning at 9 am
pleWsd finX me a trAin fEom fahbridge to stWHsted xirporh ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford .
i wili be trAveliMg on wednrsFay . ||| i wilp be travelJnF on thJrzday .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am lolkihg for a trAin f5om caJbridgd to ely .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am lolkihg for a trAin f5om caJbridgd to ely .
i Sant to go to petetgorouTh . ||| i ll be goiJg to Xakbridge on the samw day as my st0p at trabeolers resg .
i wili be trAveliMg on wednrsFay . ||| i neSd to go to stfvejage on wednRcday
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i aould l8ke it to be in northeFB cqkbridge and hace 2 starC .
i ll be goiJg to Xakbridge on the samw day as my st0p at trabeolers resg . ||| i Sant to go to petetgorouTh .
yes can you nake me a bookimv for 8 peolpe ? ||| yes ppeaee cFeWte a bookonf for 6 LSople . it needq to be for 2 niyhfs xfarting on tueddaT .
11 pm and i wang it for thrfe pePpie ||| 11 pm and i wanH it for tnree pei9le
i am 5Gying to biok a trAin to hdoxbourne . can you hWlp me ? ||| i am lolkihg for a trAin f5om caJbridgd to ely .
i ll be departlmg froj broDbournW and i d llke to go to sambridgF . ||| i wKll be gking to Vanbridge .
i wKll be gking to Vanbridge . ||| i ll be Peavimg rrom leiceeted and 1ant to go to canbridYe .
i ll be travelKnN on wedbesda5 , and i dill be go to petegbLrIugh . ||| i wili be trAveliMg on wednrsFay .
hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ? ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ?
i wili be trAveliMg on wednrsFay . ||| i wikl be trAvelong on sW6urday .
11 pm and i wanH it for tnree pei9le ||| 11 pm and i wang it for thrfe pePpie
yes , i m lolkinf for a nhseum to go to . ||| i neWd to f8nd a 6rain leavkhg Sambr8dge and goibg to ely . can you melp ?
i Sant to go to petetgorouTh . ||| i am lPokinR to go to BPrwich .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am 5Gying to biok a trAin to hdoxbourne . can you hWlp me ?
i wili be trAveliMg on wednrsFay . ||| 11 pm and i wang it for thrfe pePpie
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| yes , i jeed a trAin out of cxNbridge on tMurwday , oleaWe .
i ll be travellumg on jonRay . ||| i wKll be gking to Vanbridge .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am 5Gying to biok a trAin to hdoxbourne . can you hWlp me ?
i wamt to go to Fahbridge , pless3 . ||| i am lPokinR to go to BPrwich .
i wili be trAveliMg on wednrsFay . ||| i wilp be travelJnF on thJrzday .
hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ? ||| i nerd a trAin froH kingD l7nn to go to caNbrOdge
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford .
i Sant to go to petetgorouTh . ||| 11 pm and i wang it for thrfe pePpie
