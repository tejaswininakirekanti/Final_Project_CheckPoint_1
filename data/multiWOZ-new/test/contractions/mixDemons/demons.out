O O O O O O B-depart I-depart O B-leave ||| O O O O O B-depart I-depart O B-leave
O O O O O O B-leave O ||| O O O O O O B-leave O
O O O O O O O O O O B-dest O ||| O O O O O O O O B-dest O
O O O O O B-day O ||| O O O B-depart O B-day O
O O O O O O O O B-people O O ||| O O O O O B-people O O O
O O O O O O O O O O O B-depart I-depart I-depart O ||| O O O O O O O O O O O B-depart I-depart I-depart O
O O O O O B-day O O O O O O O B-leave O ||| O O O O O B-day O O O O O O O B-leave O
O O O O B-depart I-depart I-depart O O O B-dest O ||| O O O O B-depart O O O B-dest I-dest I-dest O
O O O O O B-day O ||| O O O O O O B-day O
O O O O O O O O O B-dest O B-day O ||| O O O O O O O O O B-dest O B-day O
O O O O O O B-dest I-dest I-dest O O O B-leave O ||| O O O O O O O O O O O O O B-leave O O O O B-depart I-depart I-depart O
O O O O O O O O O B-day O O O B-dest O ||| O O O O O O O O O O O B-day O B-dest O
O O O O O B-depart ||| O O O O O B-depart
O O O O B-day O ||| O O O O O O B-day O
O O O O O O O B-depart O B-day O ||| O O O O O O O B-depart O O B-day O
O O O O O O O O O B-dest O O O B-arrive O ||| O O O O O O O O O B-dest O O O B-arrive O
O O O O O O B-depart O ||| O O O O O O O O O B-depart O O B-day O
O O O O O O B-dest O B-day O ||| O O O O O O B-dest O B-day O
O O O O O B-depart O B-day O ||| O O O O O B-depart O B-day O
O O O O O O B-leave O ||| O O O O O O B-leave O
O O O O O O O B-depart O ||| O O O O O O O O O B-depart O
O O O O O O O O O B-dest O B-day O ||| O O O O O O O O O B-dest O B-day O
O O O O O B-dest O B-arrive O O ||| O O B-depart O O O O B-arrive O
O O O O B-depart I-depart I-depart O B-day O ||| O O O O O B-day O O O B-dest I-dest I-dest O
O O O O O O B-leave O O O ||| O O O O O O B-leave O O O
O O O O O O O B-day O ||| O O O O O O B-day O O
O O O O O B-day O O O ||| O O O O B-day O
O O O O O O O O O B-day O B-leave O ||| O O O O O O O O B-day O O O B-arrive O
O O O O B-depart O ||| O O O O B-depart O
O O O O O O O O O O B-dest I-dest I-dest O O O B-day ||| O O O O O O O B-day O O B-dest I-dest I-dest O
O O O O O B-dest O O O O O O O O B-leave O ||| O O O O O O O O B-arrive O O B-depart O
O O O B-depart I-depart I-depart O O O B-dest O ||| O O O O O O B-dest O O O O O B-depart O
O O O O O O B-dest O B-day O ||| O O O O O O B-dest O B-day O
O B-day O O O ||| O B-day O O O
O O O B-depart O B-dest O ||| O O B-depart O B-dest O
O O O O O O O O O O B-people O O ||| O O O O O O O O O O O B-people O O O
O O O O O O B-dest O ||| O O O O O O O O O O B-depart O B-day
O O O O O B-depart O O O ||| O O O O O B-depart O O O
O O O O O O O O O O O O O B-depart I-depart O B-dest O B-leave O O B-day O ||| O O O O O O O O B-day O B-leave O O B-depart I-depart O O O B-dest O
O O O O O O O B-dest I-dest I-dest O O O B-arrive O O O O O O O ||| O O O O O O O O O B-leave I-leave O O O O B-depart I-depart I-depart
O O O O O O B-day O ||| O O O O O B-day O
O O O O B-dest I-dest ||| O O O O B-dest I-dest
O O O O O O O B-day O ||| O O O O O O O O B-day O
O O O O O O B-day O O O ||| O O O O O O B-day O O O
O O O O O O O O O B-day O ||| O O O O O O O O O B-day O
O O O O O B-dest I-dest I-dest O O O O B-depart O ||| O O O O O B-dest I-dest I-dest O O O O B-depart O
O O O O O O O O B-depart O B-leave O ||| O O O O O O O O B-depart O B-leave O
O O O O B-day O ||| O O O O B-day
O O O O O B-depart O B-day O ||| O O O O B-depart O B-day O
O O O O B-dest O ||| O O O O B-dest O
O O O O O B-arrive O ||| O O O O O B-arrive
O O O O B-day O ||| O O O O O B-day
O O O O B-day O B-leave O ||| O O O O B-day B-leave I-leave O
O O O O O O B-dest O B-arrive O ||| O O O O O O B-dest O B-arrive O
O O O O O O O O O O O O B-depart O B-day O ||| O O O O O O O O O O O B-depart O B-day O O O B-dest O B-arrive O
O O O O O O B-day O O O O O O O O ||| O B-day O O O O O O O B-arrive
O O O O O O O O O O O B-arrive ||| O O O O O O B-day O O O O B-leave O
O O O O O B-depart O O B-dest O ||| O O O O O B-depart O O B-dest O
O O O O O O O O B-dest O ||| O O O O O O O B-dest O
O O O O O B-depart I-depart I-depart O ||| O O O O O B-depart O
O O O O O O B-leave O ||| O O O O O B-leave I-leave O
O O O O O O O O B-depart O B-dest O ||| O O O O O O O B-depart O B-dest O B-day O
O O O O B-day O ||| O O O O O B-day
O O O O O O B-day O O B-leave O ||| O O O O O O B-leave O
O O O O B-dest ||| O O O O O B-dest O
O O O O O B-day O O O O O O B-arrive O ||| O O O O O B-day O O O O O O B-arrive O
O O O O O O B-day O B-leave O O B-dest I-dest I-dest O ||| O O O O B-dest I-dest I-dest O O O O O O O B-day O
O O O O O B-depart O B-dest ||| O O O O O O O O O O O O O O O O O B-depart O B-dest O
O O O O O O B-day O O O O B-leave O ||| O O O O O B-day O O O O O O B-leave O
O O O O B-day O ||| O O O O O O B-day O
O O O O O O B-day O B-depart O O O B-dest O B-arrive O ||| O O O O O B-dest O O B-depart O O B-day O
O O O B-depart O B-day I-day O O O O O O B-leave O ||| O O O B-depart O B-day I-day O O O O O O B-leave O
O O O O O O O O B-leave O ||| O O O O O O O O B-leave O
O O O O O O O O O O O O O O O B-people O O O ||| O O O O O O O O O O O O O O O B-people O O
O O O O O O O O B-dest I-dest I-dest O O B-leave O ||| O O O O O O B-depart I-depart I-depart O B-leave O O O O O O O O O
O O O O O O O B-dest O O O B-leave ||| O O O O O O O B-dest O O O B-leave
O O O O O O O O B-arrive O O O ||| O O O O O O O B-arrive O B-day O
O O O O O O B-leave O O ||| O O O O O O B-leave O O
O O O O O B-depart O B-day O ||| O O O O O B-depart O B-day O
O O O O O O B-leave ||| O O O O O O B-leave
O O O O O O O O O B-depart I-depart I-depart O B-dest O ||| O O O O O O O O O O O B-depart O O O B-dest I-dest I-dest O
O O O O O O O O B-dest I-dest O ||| O O O O O O O O B-dest I-dest O
O O O O O O B-day O O ||| O O O O B-day O
O O O O O O O O O B-day O ||| O O O O O O O O O B-day O
O O O O O B-depart ||| O O O O O O O O B-dest O
O O O O B-depart O B-dest O O B-leave O ||| O O O O B-depart O B-dest O O B-leave O
O O O O O O O O O O O B-arrive O ||| O O O O O O O O O O O B-arrive O
O O O O O O O O O B-dest I-dest I-dest O ||| O O O O O O O O O O O B-depart I-depart I-depart O
O O O O O O O O O B-dest O B-arrive O ||| O O O O O O B-dest O B-arrive O
O O O O O O O O B-people O O O O O O O O ||| O O O O O O O O B-people O O O O O O O O
O O O O O O B-leave ||| O O O O O O B-leave O O
O O O O B-day O O O O B-arrive O O B-leave I-leave I-leave O O O O O O O O O O O O O ||| O O O O O O O B-arrive O O O O O O O O O O O O O
O O O O O B-depart O ||| O O O O O B-depart O
O O O O O B-depart I-depart O O O B-dest O ||| O O O O O B-depart I-depart O O O B-dest O
O O O O O B-day O ||| O O O O O B-day O
O O O O O O O B-depart O B-day O O O B-leave O ||| O O O O O O O O O B-depart O B-leave O
O O O O O O O O B-dest O O O ||| O O O O O O O O B-dest O O O
O O O O O O B-day O ||| O O O O O O B-day O
O O O O O O B-leave O O O ||| O O O O O O B-leave O O O
O O O O O B-depart O B-day O O O O O O O O O B-arrive O O ||| O O O O B-depart O O O O O O O O B-arrive O O O
O O O O B-dest O O O O O B-leave O ||| O O O O B-dest O O O O O O B-leave O
O O O O O O O O O O B-day O B-depart I-depart I-depart O ||| O O O O O O O O O O B-day O B-depart I-depart I-depart O
O O O O O O B-day O O O O B-arrive ||| O O O O O O B-day O O O B-arrive O
O O O O O O O O O O B-day O B-leave O ||| O O O O O O O O O O B-day O B-leave O
O O O O O B-depart I-depart I-depart O B-leave O ||| O O O O O B-depart I-depart I-depart O
O O O O O O O O O B-leave I-leave O O O O B-depart I-depart I-depart ||| O O O O O O O O O B-leave I-leave O O O O B-depart I-depart I-depart
O O O O O O B-depart O O O O ||| O O O O O B-depart O
O O O O O O B-day O O ||| O O O O O O B-day O
O O O O O B-depart O B-day O ||| O O O O O O O O B-depart O B-day O
O O O O O O O B-dest O O O B-depart I-depart O ||| O O O O O O O B-depart I-depart O O O B-dest O
O O O O O O B-arrive O B-dest O ||| O O O O O O O O B-arrive O O
